definition/ISetting.ts

```diff
@@ -9,8 +9,8 @@ export enum SettingEditor {
 	EXPRESSION = 'expression'
 }
 
-export type SettingValueMultiSelect = Array<{key: string; i18nLabel: string}>
-export type SettingValueRoomPick = Array<{_id: string; name: string}> | string
+export type SettingValueMultiSelect = string[];
```

Comment:
Why was this needed? 👀

Reply:
:shushing_face: 

jk :joy: the type was actually wrong.. the value for a multi-select setting is an array of strings.. so I had to fix the type to be able to use it correctly.

Reply:
here is the setting record:
```
{
    "_id" : "Hide_System_Messages",
    "createdAt" : ISODate("2021-01-11T12:21:35.320Z"),
    "group" : "Message",
    "i18nDescription" : "Hide_System_Messages_Description",
    "i18nLabel" : "Hide_System_Messages",
    "ts" : ISODate("2021-06-14T12:21:41.761Z"),
    "type" : "multiSelect",
    "value" : [ 
        "uj", 
        "ul", 
        "ru"
    ],
    "values" : [ 
        {
            "key" : "uj",
            "i18nLabel" : "Message_HideType_uj"
        }, 
        {
            "key" : "ul",
            "i18nLabel" : "Message_HideType_ul"
        }, 
        {
            "key" : "ru",
            "i18nLabel" : "Message_HideType_ru"
        }, 
        {
            "key" : "au",
            "i18nLabel" : "Message_HideType_au"
        }, 
        {
            "key" : "mute_unmute",
            "i18nLabel" : "Message_HideType_mute_unmute"
        }, 
        {
            "key" : "r",
            "i18nLabel" : "Message_HideType_r"
        }, 
        {
            "key" : "ut",
            "i18nLabel" : "Message_HideType_ut"
        }, 
        {
            "key" : "wm",
            "i18nLabel" : "Message_HideType_wm"
        }, 
        {
            "key" : "rm",
            "i18nLabel" : "Message_HideType_rm"
        }, 
        {
            "key" : "subscription-role-added",
            "i18nLabel" : "Message_HideType_subscription_role_added"
        }, 
        {
            "key" : "subscription-role-removed",
            "i18nLabel" : "Message_HideType_subscription_role_removed"
        }, 
        {
            "key" : "room_archived",
            "i18nLabel" : "Message_HideType_room_archived"
        }, 
        {
            "key" : "room_unarchived",
            "i18nLabel" : "Message_HideType_room_unarchived"
        }, 
        {
            "key" : "room_changed_privacy",
            "i18nLabel" : "Message_HideType_room_changed_privacy"
        }, 
        {
            "key" : "room_changed_avatar",
            "i18nLabel" : "Message_HideType_room_changed_avatar"
        }, 
        {
            "key" : "room_changed_topic",
            "i18nLabel" : "Message_HideType_room_changed_topic"
        }, 
        {
            "key" : "room_e2e_enabled",
            "i18nLabel" : "Message_HideType_room_enabled_encryption"
        }, 
        {
            "key" : "room_e2e_disabled",
            "i18nLabel" : "Message_HideType_room_disabled_encryption"
        }
    ]
}

Reply:
Ah I see it 😂 nice nice, let's approve this then 👀